spec/index.md

```diff
@@ -2957,9 +3005,16 @@ Finally, we can describe the state of the IC as a record having the following fi
       subnet_id : Principal;
       subnet_size : Nat;
     }
+    Snapshot = {
```

Comment:
maybe we should store the system-generated snapshot ID here so that we can check it later